stormpath/stormpath-sdk-node
# Rename test/sp.authc.apiKey.js to test/sp.authc.apiKey_test.js

Renames `test/sp.authc.apiKey.js` to `test/sp.authc.apiKey_test.js` to make the test run.

### How to verify

1. Checkout this branch
2. Run `npm test` and verify that this test is being run
